### ISSUE TITLE
Token Withdraw from contract

### ISSUE THREAD

**Issue by PCCalc** [OpenSTFoundation/SimpleTokenSale#20]
This may be the wrong place to put this but I'm at a loss.
Can tokens sent to the contract be withdrawn? I know that its an option I just don't know where to see in the code if the option is on or off.

I could really use some help on this. I made a huge mistake and sent tokens directly to this contract address.

**Comment by jasonklein** [comment]
@PCCalc: a smart contract can only undertake actions explicitly permitted by its code; I am not certain what gave you the impression that functionality exists that would enable "withdrawing" tokens transferred to the address of the `SimpleToken` contract, but the contract's code does not include functionality to transfer, or to approve transfers of, tokens transferred to it.

It is unfortunate if tokens are mistakenly transferred to the address of the token contract, as it is unfortunate if tokens are ever mistakenly transferred, but the functionality does not exist for the contract to transfer tokens on its own behalf.

**Comment by PCCalc** [comment]
Thank you for your answer it seems that it is not possible. I just couldn't tell from reading the contract code. To answer your question where I got the Idea from.
https://ethereum.stackexchange.com/questions/34559/i-accidentally-sent-a-token-to-a-token-contract-can-i-get-it-back
https://theethereum.wiki/w/index.php/ERC20_Token_Standard

I just wasn't sure if it was implemented on this contract and thought it would be easier to ask then to try and decipher it my self. 
Thank you again.
